pkg/ebpf/c/common/hash.h

```diff
@@ -94,4 +94,17 @@ u32 hash_u32_and_u64(u32 arg1, u64 arg2)
     return murmur32(buffer, 4 + 8); // 4 + 8 = sizeof(u32) + sizeof(u64)
 }
 
+// This function is a wrapper for regular hash_u32_and_u64 function. The difference is that it knows
+// u64 argument is a timestamp for a task "starttime" and it will round up the timestamp to the
+// maximum precision userland can have (so both, eBPF and userland, are able to calculate the same
+// hash for the same task).
+
+u32 hash_u32_and_roundup_u64(u32 arg1, u64 arg2)
+{
+    u64 round = arg2 / 10000000LL;
```

Comment:
I tested it and it looked okay. Thats actually how I discovered the starttime from userland had a smaller precision. Since hash is pid + starttime. The pid won't repeat until the next "full pid number cycle", which will take much longer than anything close to the rounding number, I think we're safe. If we aren't (And discover) we can always fix this.